.github/workflows/ci.yml

```
@@ -292,16 +297,18 @@ jobs:
       - name: "Checking out runtime submodules"
         run: ./build_tools/scripts/git/update_runtime_submodules.sh
       - name: "Building runtime"
+        # Note ccache is read-only here since GitHub runners don't have write
+        # access to GCS
```

This comment lacked some context / wasn't clear to me at first (you'd need to recognize that `runs-on: ubuntu-20.04-64core` is GitHub-hosted, compared to `runs-on: managed-windows-cpu` being self-hosted)

```suggestion
        # Note ccache is read-only here since this job runs on a GitHub-hosted runner
        # and GitHub runners don't have write access to GCS
```

---

managed-windows-cpu is also GitHub hosted

---

(but clarification still makes sense and I applied the fix)